### PR TITLE
AP_ExternalAHRS: Fixed InertialLabs External AHRS driver

### PR DESCRIPTION
**Pull request by VSheleverIL**
1. According to the Inertial Labs INS documentation, for conversion accelerometer output data, the value of g = 9.8106 m/s should be used. Using the GRAVITY_MSS constant will increase accelerometer errors.
https://github.com/ArduPilot/ardupilot/blob/ae19d032ce73fdbf4eb7c375079244131b24babb/libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp#L270
2. The differential pressure in the Inertial Labs INS data format is represented in mbar. Therefore, for use in the ArduPilot algorithm, the value must be converted to Pa (Pascal).
https://github.com/ArduPilot/ardupilot/blob/ae19d032ce73fdbf4eb7c375079244131b24babb/libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp#L381
3. Using the offset_bearing() function to calculate northing and easting speeds is incorrect in this case, so a direct calculation has been added.
https://github.com/ArduPilot/ardupilot/blob/ae19d032ce73fdbf4eb7c375079244131b24babb/libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp#L353
4. Following the recommendations of Inertial Labs, the output data format (data set) has been expanded to include the following data: 
    - Dilution of Precision (there is used in the algorithm);
    - GNSS Heading, GNSS Pitch, GNSS Angles position type, GNSS Heading timestamp (сurrently not implemented in the ArduPilot algorithm.). 

   Additionally, logging of these data types has been added.

### CODE REVIEW

File: libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp [hunk 1]
```diff
@@ -466,17 +490,13 @@ bool AP_ExternalAHRS_InertialLabs::check_uart()
             last_gps_ms = now_ms;
         }
     }
-#if AP_BARO_EXTERNALAHRS_ENABLED
     if (GOT_MSG(BARO_DATA) &&
         GOT_MSG(TEMPERATURE)) {
         AP::baro().handle_external(baro_data);
     }
-#endif
```
Comment (rmackay9) [inline libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp:474]: Removing these definitions doesn't look right.  I suspect this change was unintentional.

File: libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp [hunk 2]
```diff
@@ -267,7 +268,7 @@ bool AP_ExternalAHRS_InertialLabs::check_uart()
         }
         case MessageType::ACCEL_DATA_HR: {
             CHECK_SIZE(u.accel_data_hr);
-            ins_data.accel = u.accel_data_hr.tofloat().rfu_to_frd()*GRAVITY_MSS*1.0e-6;
+            ins_data.accel = u.accel_data_hr.tofloat().rfu_to_frd()*9.8106f*1.0e-6;
```
Comment (rmackay9) [inline libraries/AP_ExternalAHRS/AP_ExternalAHRS_InertialLabs.cpp:271]: nitpick: white space at the end.

**Review by rmackay9 — CHANGES_REQUESTED** [review]
Hi @VSheleverIL,

Thanks for this.  Could you correct the commit title so that it starts with "AP_ExternalAHRS:"?  If you look through [our commit history](https://github.com/ArduPilot/ardupilot/commits/master/) you'll see we always prefix our commits with the subsystem to make it easier to see what is affected by the commit.  This is especially important during backports.  To be clear I meant the commit title, not the PR title.

Could you also fill in the PR description including testing that has been performed?

**Comment by VSheleverIL** [comment]
> Hi @VSheleverIL,
> 
> Thanks for this. Could you correct the commit title so that it starts with "AP_ExternalAHRS:"? If you look through [our commit history](https://github.com/ArduPilot/ardupilot/commits/master/) you'll see we always prefix our commits with the subsystem to make it easier to see what is affected by the commit. This is especially important during backports. To be clear I meant the commit title, not the PR title.
> 
> Could you also fill in the PR description including testing that has been performed?

**Comment by VSheleverIL** [comment]
> Hi @VSheleverIL,
> 
> Thanks for this. Could you correct the commit title so that it starts with "AP_ExternalAHRS:"? If you look through [our commit history](https://github.com/ArduPilot/ardupilot/commits/master/) you'll see we always prefix our commits with the subsystem to make it easier to see what is affected by the commit. This is especially important during backports. To be clear I meant the commit title, not the PR title.
> 
> Could you also fill in the PR description including testing that has been performed?

Hi @rmackay9,

Thank you for your feedback. I have made changes according to your comments. Please review the outcome. Thank you!

**Pull request closed without merging**